.semaphore/semaphore.yml.erb

```diff
@@ -0,0 +1,71 @@
+<%
```

Comment:
How do I use this file to generate the other file? 
I feel like a wrapper Ruby script is missing and some instructions on how to generate it (see comment about adding the warning to `semaphore.yml`).

Reply:
I thought it would be `erb .semaphore/semaphore.yml.erb`, but that's not working for me as-is on Ruby 3.0.2 or Ruby 2.7.5. [For some reason I get some syntax errors due to the `-` in `<% something -%>` to remove trailing whitespace 🤔 and removing those makes it work? No idea why this is happening, I'm pretty sure those are valid syntax]

Reply:
erb -T- .semaphore/semaphore.yml.erb > .semaphore/semaphore.yml

Reply:
A Ruby script is not “missing”, I chose not to add a script here because it’s a single command. Also, we’re currently not using Ruby in this repository and I’d prefer to keep it that way. Would adding a Make command to wrap this line be good enough?

Reply:
Huh, that works, thanks! The `-T-` thing is non-obvious, so I agree with Tom here -- a wrapper (a Rakefile, or even a shell script) would be neat.

Reply:
A wrapper script would be nice. Personally, not a fan of Makefiles for small things like this, [a `script/<name>` naming scheme](https://github.com/github/scripts-to-rule-them-all) like we do in all other integrations repositories would be good.

Like Noemi pointed out, a step in the CI to validate you're running what you think you've confirmed is a must have to avoid headaches later on.